src/pages/manage/manage.html

```diff
@@ -96,11 +101,11 @@
           <ion-card-content>
             <ion-label stacked>Paypal Email Address</ion-label>
             <ion-input
-                    type="email"
-                    placeholder="example@example.com"
-                    required
-                    name="email"
-                    [(ngModel)]="withdrawMoney.email"
+              type="email"
+              placeholder="example@example.com"
+              required
+              name="email"
+              [(ngModel)]="withdrawMoney.email"
             ></ion-input>
```

Comment:
You should use [forms with FormBuilder](https://ionicframework.com/docs/v2/resources/forms/). Its easier to test and cleaner. Its also consistent with the rest of our code.